ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-standard-ranGen.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "group-du"
```

Comment:
duplicated name as group-du-3node-ranGen.yaml .

Reply:
Good catch!  I just found I had to introduce a differentiation between the 3-node and standard clusters.  I'll rename to match.

Reply:
Done.